prisma/schema.prisma

```diff
@@ -91,6 +91,8 @@ model Post {
   tags        Tag[]
   authorId    String
   author      User      @relation(fields: [authorId], references: [id], onDelete: Cascade)
+  // deletedAt   DateTime?
+  // totalLike   Int       @default(0)
```

Comment:
thường không làm nữa thì delete hẳn, để comment đôi khi phiền lắm vì không ai dám xoá, lâu dần toàn rác